### PR TITLE
add product reference to bundle

### PR DESCRIPTION
**Pull request by michz**

### CODE REVIEW

File: src/Entity/ProductBundle.php [hunk 1]
```diff
@@ -24,6 +24,9 @@ class ProductBundle implements ProductBundleInterface, ResourceInterface, CodeAw
     /** @var Collection */
     private $products;
 
+    /** @var ProductInterface */
+    private $product;
```
Comment (mattagohni) [inline src/Entity/ProductBundle.php:28]: Not sure if we will later be aware that this product represents the product-entity of the product-bundle-entity. Maybe there is a good chance to be confused with $product and $products...

Comment (michz) [reply src/Entity/ProductBundle.php:28]: I had the same thoughts but no idea how to name it better. Suggestions?

Comment (michz) [reply src/Entity/ProductBundle.php:28]: Suggestion: Rename "products" to bundledProducts" and leave "product" as it is

Comment (mattagohni) [reply src/Entity/ProductBundle.php:28]: bundleProducts and product would be ok for me. 

Btw maybe we should not assign Products directly to the Bundle, somehow we need to know which product we are looking at. E.g. 'do we have a T-Shirt or an Short'. So I thought about doing the relation between bundles and its products through an additional resource, e,g. 'ProductBundleSlots'.

Comment (michz) [reply src/Entity/ProductBundle.php:28]: After verbal discussion: Current ```$products``` will be removed in future, so this is not relevant by now.

**Review by mattagohni — APPROVED** [review]

**Review by sd-lueckel — APPROVED** [review]

**Pull request merged**